src/main/java/edu/utdallas/davisbase/host/Host.java

```diff
@@ -39,9 +42,68 @@ public Host(HostConfiguration configuration, BufferedReader reader, PrintWriter
     this.writer = writer;
   }
 
-  public String readStatement() throws IOException {
-    // TODO Implement Host.readStatement()
-    throw new NotImplementedException();
+  static String prompt = "davisql> ";
+	static String copyright = "Team Blue";
+  static String version = "v1.0";
+  static boolean exit_prompt = false;
```

Comment:
Remove this. It's not used. And mutable `static` variables are almost always bad practice anyway. (it is here)